module.go

```diff
@@ -147,7 +147,7 @@ func (m *module) provide(p provide) {
 	}
 
 	var info dig.ProvideInfo
-	if err := runProvide(m.scope, p, dig.FillProvideInfo(&info), dig.Export(true)); err != nil {
+	if err := runProvide(m.scope, p, dig.FillProvideInfo(&info), dig.Export(!p.Private)); err != nil {
```

Comment:
This comment is not related to this line at all, but maybe it's worth adding `Private` field to `fxevent.Provided` to indicate a Provide was private in the event?

Reply:
Also, if we do this, maybe we should update ZapLogger and ConsoleLogger implementation as well for logging Provided events

Reply:
Hmm good idea, let me know if you have thoughts on how I added it to `ConsoleLogger`